packages/app/src/composables/useTestsForDebug.ts

```diff
@@ -0,0 +1,26 @@
+import { gql, useQuery } from '@urql/vue'
+import { Debug_TestsForDebugDocument } from '@packages/app/src/generated/graphql'
+import { computed } from 'vue'
+
+gql`
+  query Debug_TestsForDebug($specTitle: String!) {
+    currentProject {
+      id
+      testsForReviewBySpec(specTitle: $specTitle)
+    }
+  }
+`
+
+export function useTestsForDebug () {
+  const variables = { specTitle: '' }
+
+  const query = useQuery({ query: Debug_TestsForDebugDocument, variables, pause: true })
```

Comment:
This is very clunky compared to a mutation, but I don't see any alternative other than using a mutation.

Reply:
This is just the way you have to do it.